libcache/cache.c

```diff
@@ -101,16 +101,16 @@ static uint64_t cache_generateMask(int numBits)
 }
 
 
-cachectx_t *cache_init(size_t srcMemSize, size_t size, size_t lineSize, const cache_ops_t *ops)
+cachectx_t *cache_init(size_t srcMemSize, size_t numLines, size_t lineSize, const cache_ops_t *ops)
```

Comment:
For consistency sake (object should be first, like lineSize and srcMemSize)
```suggestion
cachectx_t *cache_init(size_t srcMemSize, size_t linesCnt, size_t lineSize, const cache_ops_t *ops)
```